frontend/src/pages/CardPage/components/Slider/Slider.jsx

```diff
@@ -1,63 +1,120 @@
-import React from 'react';
+/* eslint-disable max-len */
+/* eslint-disable import/no-unresolved */
+import React, { useRef, useEffect } from 'react';
 import styles from './Slider.module.scss';
+import './swiper.css';
 import IconButton from '../../../../components/ui-kit/components/IconButton';
 import { ICONS } from '../../../../components/ui-kit/icons';
 import Card from '../../../../components/common/Card/Card';
+import { register } from 'swiper/element/bundle';
+
+const Slider = React.memo(({ data }) => {
+  const swiperElRef = useRef(null);
+
+  useEffect(() => {
+    register();
+
+    const params = {
+      spaceBetween: 20,
+      breakpoints: {
+        320: {
+          slidesPerView: 2,
+          spaceBetween: 8,
+        },
+        510: {
+          slidesPerView: 3,
+        },
+        710: {
+          slidesPerView: 4,
+        },
+        868: {
+          slidesPerView: 3,
+        },
+        1180: {
+          slidesPerView: 4,
+          navigation: {
+            nextEl: '.swiper-button-next',
+            prevEl: '.swiper-button-prev',
+          },
+          pagination: {
+            el: '.swiper-pagination',
+            clickable: true,
+            renderBullet: function (index, className) {
+              return (
+                '<span class="' +
+                className +
+                ' ' +
+                styles.paginationBullet +
+                '">' +
+                '</span>'
+              );
+            },
+          },
+        },
+      },
+      injectStyles: [
```

Comment:
we do not use inline styles

Reply:
I need it for overwriting initial swiper styles